person.rb

```diff
@@ -1,11 +1,11 @@
 require './nameable'
 
-class Person
-  include Nameable
+class Person < Nameable
   attr_accessor :name, :age
   attr_reader :id
 
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you make your pull request title more descriptive and professional 👍 by replacing `Add content to file` with `OOP school library: decorate a class` to make your pull request professional 😊 Your pull request title should tell us what is happening in this pull request 🤔
- [ ] It would be nice if you add some more details in your pull request summary to make your pull request professional :+1: Your pull request should have all the necessary details of what you are doing in this pull request :blush: